src/main/rx/cc2500_frsky_shared.c

```diff
@@ -513,7 +514,7 @@ static bool frSkySpiDetect(void)
     return false;
 }
 
-bool frSkySpiInit(const rxConfig_t *rxConfig, rxRuntimeConfig_t *rxRuntimeConfig)
+bool frSkySpiInit(const rxSpiConfig_t *rxSpiConfig, rxRuntimeConfig_t *rxRuntimeConfig)
```

Comment:
These changes need to be made in `rx/flysky.c` as well. (Test with CRAZYBEEF3FS.)